index.bs

```diff
@@ -209,83 +249,81 @@ enum OperandType {
   "uint8"
 };
 
-dictionary OperandDescriptor {
+dictionary MLOperandDescriptor {
   // The operand type.
-  required OperandType type;
+  required MLOperandType type;
 
   // The dimensions field is only required for tensor operands.
   // The negative value means an unknown dimension.
   sequence<long> dimensions;
 };
 </script>
 
-## Operand ## {#api-operand}
+## MLOperand ## {#api-mloperand}
 <script type=idl>
 [SecureContext, Exposed=Window]
-interface Operand {};
+interface MLOperand {};
 </script>
 
-## NeuralNetworkContext ## {#api-neuralnetworkcontext}
+## MLModelBuilder ## {#api-mlmodelbuilder}
 
-The {{NeuralNetworkContext}} interface represents a global state of neural network compute workload and execution processes.
+The {{MLModelBuilder}} interface defines a set of operations as identified by the [[#usecases]] that can be composed into a computational graph. It also represents the intermediate state of a graph building session.
 
 <script type=idl>
-[SecureContext, Exposed=Window]
-interface NeuralNetworkContext {
-  ModelBuilder createModelBuilder();
-};
-</script>
-
-## ModelBuilder ## {#api-modelbuilder}
+typedef record<DOMString, MLOperand> MLNamedOperands;
 
-The {{ModelBuilder}} interface defines a set of operations as identified by the [[#usecases]] that can be composed into a computational graph. It also represents the intermediate state of a graph building session.
-
-<script type=idl>
-typedef record<DOMString, Operand> NamedOperands;
+dictionary MLBuffer {
+  required (ArrayBufferView or WebGLBuffer or GPUBuffer) resource;
```

Comment:
Should we use `ArrayBuffer` instead of `ArrayBufferView`? The `offset` and `size` would means byte offset and size that align to WebGPUBuffer.

Reply:
If I'm not mistaken, the offset and length of the `ArrayBuffer` is defined as 32-bit integer while those of the GPU types are 64 bits.

Reply:
I am not sure whether the 32-bit integer length is an implementation behavior. But regarding to [`AllocateArrayBuffer`](https://tc39.es/ecma262/#sec-allocatearraybuffer) of ecma262 spec, the `byteLength` is not restricted to 32-bit integer. And the V8 seems to define `maxByteLength` of `ArrayBuffer` to [`Number.MAX_SAFE_INTEGER`](https://tc39.es/ecma262/#sec-number.max_safe_integer) (2^53 - 1) on 64-bit systems.